### PR TITLE
Namespace output directory

### PR DESCRIPTION
**Pull request by Tjoosten**
**note**: needs to be tested. and this is a baby step to psr-2

### CODE REVIEW

**Comment by pietercolpaert** [comment]
I refuse to merge pull requests that you haven't tested. Just running the code is not that difficult:
```
# 1. clone the repo and composer install it
composer install
# 2. run a PHP test server
php -S localhost:8080 -t api
# go to e.g., http://localhost:8080/vehicle.php?id=ice10 and check whether your browser still shows results
```

**Comment by Tjoosten** [comment]
Ok sorry

**Comment by pietercolpaert** [comment]
I'm not sure whether we should namespace this "legacy" code as I wouldn't recommend anyone to reuse this code. It's just here because otherwise the API breaks.

I would like to slowly move all the parsing and querying NMBS/SNCB website in a namespaced directory however and namespace it using irail\scraper. The legacy code can then use this scraper to build the HTTP messages, as if it were any package. I'm going to wait doing that until the NMBS/SNCB has communicated in public about their data sharing policy, as they'd start sharing data starting one of these days...

Long story short: I'm going to close this pull request as I don't want to make this legacy code reusable.
